projects/canopy/src/lib/icon/docs/guide.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/UI icon/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# UI icon
+
+<p class="standfirst">UI icons are visual depictions of an object, action or idea. They're simple and easy to recognise, and can communicate a lot with very little.</p>
+
+![example](docs/ui-icon/example.png)
```

Comment:
These paths look incorrect; is there a script that maps them to the correct path on compilation?

Reply:
The path is relative to assets/ folder.

Reply:
I've explained this to @ashB100 but not as in depth as I could have so probably better for me to reply. 

The path is correct; Storybook has some [configuration](https://github.com/Legal-and-General/canopy/blob/master/.storybook/main.js#L84) to set the [static directories](https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook-configuration) to serve. Whatever is inside the folders specified in the `staticDirs` will be made available at `/` of the deployed environment.